OpenSim/Tests/ControllerExample/testControllerExample.cpp

```diff
@@ -32,10 +32,16 @@
 using namespace OpenSim;
 using namespace std;
 
+void changeVersionNumber(const std::string& filenameOld,
+                         const std::string& filenameNew);
```

Comment:
Instead of having to convert in the tests, some alternate approaches:
 1. have the option to force the version number on write (Manager and Tools must be updated to provide the version #) 
 2. write STOs of type `double` as version 1 by default and only non doubles as version 2
 3. add the conversion code to shared testing utilities 

My preference would be 1 or 2.